samcli/lib/build/build_graph.py

```diff
@@ -200,15 +204,15 @@ def get_function_build_definitions(self) -> Tuple["FunctionBuildDefinition", ...
     def get_layer_build_definitions(self) -> Tuple["LayerBuildDefinition", ...]:
         return tuple(self._layer_build_definitions)
 
-    def get_function_build_definition_with_logical_id(
-        self, function_logial_id: str
+    def get_function_build_definition_with_full_path(
+        self, function_full_path: str
     ) -> Optional["FunctionBuildDefinition"]:
         """
         Returns FunctionBuildDefinition instance of given function logical id.
 
         Parameters
         ----------
-        function_logial_id : str
+        function_full_path : str
             Function logical id that will be searched in the function build definitions
```

Comment:
nit: update the param description as well

Reply:
updated